### PR TITLE
Completed css-grid-classes. | Vera Yeh

### PR DESCRIPTION
**Pull request by VeraYeh**
image

### CODE REVIEW

File: css-grid-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    <title>CSS Grid Classes</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <header><h1>Choose your character!</h1></header>
+      </div>
+      <div class="row">
+        <div>
```
Comment (BrettAlbright) [inline css-grid-classes/index.html]: Make sure to update each of the `row` direct children with their proper column classes

File: css-grid-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+}
+
+.column {
```
Comment (BrettAlbright) [inline css-grid-classes/styles.css]: this is good for the `column-one-fourth` class, make sure to create classes for the rest of them.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
some CSS and HTML updates

**Comment by VeraYeh** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**